paraatm/tests/test_para_atm.py

```diff
@@ -112,6 +113,22 @@ def test_vcas(self):
         sim = VCAS(cfg)
         track = sim()
         self.assertEqual(len(track), 1000)
+    def test_aviationr(self):
+        cur_dir = os.path.dirname(os.path.abspath(__file__))
+        data_dir = os.path.join(cur_dir, '..', 'sample_data/')
+
+        cfg = {'fp_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_GateToGate.trx',  # flight plan file
+               'mfl_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_mfl.trx',  # mfl file
+               'data_file': data_dir + 'aviationR/data/',
+               'model_file': data_dir + 'aviationR/model/',
+               'sim_time': 1000}  # total simulation time
+
+        # call
+        sim = AviationRisk(cfg)
+        device = torch.device('cpu')
+        if torch.cuda.is_available():
+            device = torch.device('cuda')
```

Comment:
This can be removed now that you have default handling for `simulation(device=None)`.  Also you can remove `import torch` from this file.

Reply:
Got it. It has been fixed

Reply:
Thanks.  Please also remove `import torch` from `test_para_atm.py`, as it is no longer needed.

I will do some testing, but we may need to move your unit tests into a different file.  We need to make sure `test_para_atm.py` will run even for users that don't have all of your dependencies installed, and that may not work unless your unit tests are separated out.